.changeset/plenty-owls-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/db": minor
```

Comment:
Made `minor` since this renames an `exports` entry to `runtime/virtual.js`. This export is undocumented and is only used by Astro DB internally (not Astro core). Just being extra cautious with versioning.